File: app/src/at-myhuynh/java/asiantech/internship/summer/java_core/exoop/Square.java
package asiantech.internship.summer.java_core.exoop;

public class Square extends Shape {

    private int edge;

    public Square(int edge) {
        this.edge = edge;
    }

    @Override
    public double perimeter() {
        return edge * 4;
    }

    @Override
    public double area() {
        return edge * edge;
    }

    @Override
    public String print() {
        return "HV";
    }
}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/exoop/Square.kt
package asiantech.internship.summer.kotlin.exoop

class Square(private val edge: Int) : Shape() {

    override fun perimeter(): Double {
        return (edge * 4).toDouble()
    }

    override fun area(): Double {
        return (edge * edge).toDouble()
    }

    override fun print(): String {
        return "HV"
    }
}
